backend/src/backend/app/settings.py

```
@@ -0,0 +1,28 @@
+from functools import lru_cache
+from typing import Optional
```

Ruff (0.8.0)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)